src/Sdk/PasswordlessClient.cs

```diff
@@ -108,6 +111,16 @@ public async Task<UsersCount> GetUsersCount()
         return (await _client.GetFromJsonAsync<UsersCount>("users/count"))!;
     }
 
+    private string DebuggerToString()
+    {
+        if (_client.DefaultRequestHeaders.TryGetValues("ApiSecret", out var values))
+        {
+            return $"ApiUrl = {_client.BaseAddress} ApiSecret = {values.First()}";

```

Comment:
Love the addition of DebuggerToString, put lets only show last 4 chars of ApiSecret, like so `ApiSecret = ***xc45`.